iati/core/rulesets.py

```diff
@@ -261,39 +331,41 @@ def is_valid_for(self, dataset):
 
```

Comment:
`None` and `False` are undocumented return values.

Reply:
done